consensus/merge/src/test/java/org/hyperledger/besu/consensus/merge/blockcreation/MergeGenesisConfigHelper.java

```diff
@@ -60,7 +61,7 @@ default ProtocolSchedule getMergeProtocolSchedule() {
         MiningConfiguration.MINING_DISABLED,
         new BadBlockManager(),
         false,
-        false,
+        ImmutableBalConfiguration.builder().build(),
```

Comment:
I think we can somewhere have BalConfiguration.BAL_DISABLED instead of creating a new one at several places

Reply:
I would understand having `BalConfiguration.DEFAULT`, but `BalConfiguration.BAL_DISABLED` seems confusing becuase BALs are not something that can enabled or disabled via this configuration.